AppCenterDistribute/AppCenterDistributeTests/MSACDistributeUtilTests.m

```diff
@@ -262,4 +262,39 @@ - (void)testCompareReleaseSameShortVersionsDifferentVersions {
   [parserMock stopMocking];
 }
 
+- (void)testIsValidUpdateTrackPulic {
```

Comment:
Let's combine all these test functions into one and call it like `testUpdateTrack`.